flyteidl/protos/flyteidl/admin/agent.proto

```diff
@@ -21,6 +20,12 @@ enum State {
   SUCCEEDED = 4;
 }
 
+// TaskOverrides is a set of overrides that can be applied to a task.
+message TaskOverrides {
```

Comment:
did you not want to use this - https://github.com/flyteorg/flyte/blob/bcb7abef1e3f578511d558bd01e8fed8924b536e/flyteidl/protos/flyteidl/core/workflow.proto#L279 just to keep in separate and name clean? (I agree)